### PR TITLE
Ios/ios 1.1

### PR DESCRIPTION
**Pull request by rebbdif**

### CODE REVIEW

File: Pave-ios/Pave/Charts/ChartViewController.swift [hunk 1]
```diff
@@ -13,6 +13,32 @@ struct DashboardValue {
 /// View controller that manages graph
 class ChartViewController: UIViewController {
 	
+	// MARK: - Public
+	
+	/// Use this method to set values you want to display in the graph
+	func setValues(_ bars: [DashboardValue]) {
+		// todo: add context here
```
Comment (lennonc) [inline Pave-ios/Pave/Charts/ChartViewController.swift:20]: I think we can remove this comment here.

Comment (rebbdif) [reply Pave-ios/Pave/Charts/ChartViewController.swift:20]: right, forgot to remove it

File: Pave-ios/README.md [hunk 2]
```diff
@@ -22,6 +30,8 @@ More on MVP https://medium.com/omisoft/lightweight-mvp-architecture-in-ios-a16b3
 For the implementation of network requests to API, see **NetworkService**.
 
 You can see dashboard implementation in **Dashboard** folder. 
-Everything you need for building a chart is in **Charts** folder. GraphCell is a container that contains **ChartViewController** that works with the chart. 
+
+Everything you need for building a chart is in **Charts** folder. **GraphCell** is a container that contains **ChartViewController** that works with the chart. 
+To set values to ChartViewController, map your values to array of `DashboardValue`, and then pass them in `setValues` method.
```
[the remark this answers is not in the document]
Comment (rebbdif) [reply Pave-ios/README.md:35]: this explanation is in the code. I believe that it's better to leave it only in the code, that's why:
1) developers will easily find it here
2) if we put it in readme, we will duplicate the source of truth. code will change, readme may not

**Review by lennonc — APPROVED** [review]
Looks good to merge

**Comment by rebbdif** [comment]
@lennonc I don't have merge rights. Could you do it?

**Pull request merged**